buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/IntellijPlugin.kt

```diff
@@ -15,6 +15,9 @@ fun Project.configureIntellijPlugin() {
 
   val isCI: Boolean by rootProject.extra
   val jetbrainsMarketplaceToken: String? by rootProject.extra
+  val pluginSignPrivateKey: String? by rootProject.extra
```

Comment:
Pls also document in CONTRIBUTING.md how to generate such keys.

Also, does CircleCI have some CLI that would allow for automatically updating the env var without going into web settings? That'd be far less error-prone

Reply:
> automatically updating the env var without going into web settings 
You mean automatic generation of the keys and ascribing them to the corresponding env vars? Perhaps in a Docker container. Otherwise, through injection with CircleCI API, they would **not** be treated as sensitive data.

Reply:
> Otherwise, through injection with CircleCI API,

Okay... indeed, what's possible via this API? just setting **non-secret** env vars for a project? pls provide a link to their docs preferably

Reply:
https://circleci.com/docs/env-vars#injecting-environment-variables-with-api-v2

Reply:
Okay, let's just provide a link to CI project settings in CONTRIBUTING.md then